### PR TITLE
Add some examples to xnat-init

### LINKED ISSUE #12: Add an example to `init` docs for anonymous access

### PR DESCRIPTION
**Pull request by mslw**
Closes #12

### CODE REVIEW

File: datalad_xnat/init.py [hunk 1]
```diff
@@ -47,6 +47,12 @@ class Init(Interface):
             text='Initialize a dataset in the current directory',
             code_cmd='datalad xnat-init http://central.xnat.org:8080',
             code_py='xnat_init("http://central.xnat.org:8080")'),
+        dict(
+            text='Initialise with anonymous access (no credentials used)',
+            code_cmd=('datalad xnat-init https://central.xnat.org:8080 '
+                      '--credential anonymous'),
+            code_py=('xnat_init("https://central.xnat.org:8080", '
```
Comment (mih) [inline datalad_xnat/init.py:54]:
```suggestion
            code_py=('xnat_init("https://central.xnat.org", '
```

analog of above

**Review by mih — DISMISSED** [review]
Thx! I left minor edit suggestions.

**Review by mih — APPROVED** [review]
Nice, thx!

**Comment by adswa** [comment]
hey @all-contributors please add @mslw for doc, code

**Pull request merged**